@navikt/core/react/src/tooltip/Tooltip.tsx

```diff
@@ -80,6 +81,13 @@ export interface TooltipProps extends HTMLAttributes<HTMLDivElement> {
    * List of Keyboard-keys for shortcuts.
    */
   keys?: string[];
+  /**
+   * When false, Tooltip labels the element, and child-elements content will be ignored by screen-readers.
+   * When true, content is added as additional information to the child element.
+   * Important: This will default to false in future major-versions
+   * @default true
+   */
+  describeChild?: boolean;
```

Comment:
Is "describe**s**Child" clearer? (Added "s")

Reply:
I notice the comment for this will then have lied about the future
"This will default to false in future major-versions" (so we should remove that 😅 )